src/sinks/clickhouse/native/sink.rs

```diff
@@ -75,8 +95,36 @@ impl NativeClickhouseSink {
 impl StreamSink<Event> for NativeClickhouseSink {
     async fn run(self: Box<Self>, input: BoxStream<'_, Event>) -> Result<(), ()> {
         input
-            .map(|e| e.into_log())
-            .batched(self.batch.into_byte_size_config())
+            //.map(|e| e.into_log())
```

Comment:
```suggestion
```

Reply:
Oh yeah, sorry - overlooked this dev-only stuff.

Reply:
Fixed.